scripts/generate_party_positions.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+include_once dirname(__FILE__) . '/../www/includes/easyparliament/init.php';
+
+$parties = MySociety\TheyWorkForYou\Party::getParties();
+$policies = new MySociety\TheyWorkForYou\Policies;
+
+$party_count = 0;
+foreach ( $parties as $party ) {
+    if ( !$party ) {
+        continue;
```

Comment:
I think this check should be in getParties. However, I think this check is needed for: Independents, potentially Crossbench, and I guess it needs to treat Labour+Labour/Co-operative as one.